.github/workflows/docker.yml

```
@@ -0,0 +1,101 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - ci/*
```

testing?

---

You're right! The docker builds were not triggered on `ci/*` in CircleCI. They were only triggered on:
- push to master (channel=master)
- push to release/* (dry run)
- tag push (channel=candidate or channel=stable)
- schedule (channel=nightly)

Let's replace the ci/* push trigger with master push trigger that we skipped.

```suggestion
      - master
```